### ISSUE TITLE
Remote URL usage

### ISSUE THREAD

**Issue by midzer** [dimpu/angular2-markdown#45]
Hi.

It appears instructions to use markdown differ from README and Website:

`<div Markdown [path]="/path/to/readme.md"></div>
<markdown [path]="/path/to/code.cpp"></markdown>
<markdown [path]="/path/to/code.java"></markdown>`
versus
`<markdown [path]="'/assets/files/demo.md'"> </markdown>
<div Markdown [path]="'/assets/files/demo.html'"> </div>
<div Markdown [path]="'/assets/files/demo.cpp'"> </div>
<div Markdown [path]="'/assets/files/demo.java'"> </div>`

Either way my files are parsed but dont show translation from markdown language to HTML (i.e. # to <h1>). I followed tutorial at https://github.com/dimpu/angular2-markdown/blob/master/docs/getting-started/ng-cli.md carefully and there are no errors.

Cheers,
midzer

**Comment by dimpu** [comment]
Could you create a plunkr

**Comment by midzer** [comment]
Thanks for your answer, @dimpu.

In my minimal example `<markdown [path]="'assets/markdown/index.md'"></markdown>` loads file without error but formatting is not applied.

On the contrary `<div Markdown [path]="assets/markdown/index.md'"></div>` gives following error message in chromium console:

> Unhandled Promise rejection: Template parse errors:
Parser Error: Unexpected token 'Lexer Error: Unterminated quote at column 25 in expression [assets/markdown/index.md']' at column 26 in [assets/markdown/index.md'] in ng:///AppModule/ScreenComponent.html@6:16 ("

**Comment by dimpu** [comment]
It's clearly syntax error

it should be `<div Markdown [path]="'assets/markdown/index.md'"></div>`

**Comment by midzer** [comment]
Yes, you're right. Now Markdown -> HTML conversion is functional, too. I copied content from a wiki and thought it is proper markdown.

Thanks for helping out, @dimpu.
